ext/openssl/tests/ecc.phpt

```diff
@@ -0,0 +1,112 @@
+--TEST--
+openssl_*() with OPENSSL_KEYTYPE_EC
+--SKIPIF--
+<?php if (!extension_loaded("openssl") && !defined("OPENSSL_KEYTYPE_EC")) print "skip"; ?>
+--FILE--
+<?php
+
+$args = array(
+  "curve_name" => "secp384r1",
```

Comment:
Can we use tabs for indent in tests? I know that it's a bit messy and the tests have mixed indent but it would be great to use just tabs. I will try to fix other tests indent later in the same way as I did in json ext... So please just tabs in all tests...